client/reader/stream/recommended-posts.jsx

```diff
@@ -12,6 +12,24 @@ import { RelatedPostCard } from 'blocks/reader-related-card-v2';
 import PostStore from 'lib/feed-post-store';
 import Gridicon from 'components/gridicon';
 import * as stats from 'reader/stats';
+import Button from 'components/button';
+import { dismissPost } from 'lib/feed-stream-store/actions';
+
+function dismissRecommendation( uiIndex, storeId, post ) {
+	stats.recordTrackForPost( 'calypso_reader_in_stream_rec_dismissed', post, { ui_position: uiIndex } );
+	stats.recordAction( 'in_stream_rec_dismiss' );
+	dismissPost( storeId, post );
+}
+
+function handleSiteClick( uiIndex, post ) {
+	stats.recordTrackForPost( 'calypso_reader_in_stream_rec_site_clicked', post, { ui_position: uiIndex } );
```

Comment:
I made some changes to how we handle recording in-stream-rec in https://github.com/Automattic/wp-calypso/pull/9864